src/manager/mod.rs

```diff
@@ -0,0 +1,59 @@
+mod lotus_manager;
+
+use std::collections::HashMap;
+
+///! IPC node-specific traits.
+use anyhow::Result;
+use async_trait::async_trait;
+use fvm_shared::{address::Address, econ::TokenAmount};
+use ipc_gateway::Checkpoint;
+use ipc_sdk::subnet_id::SubnetID;
+use ipc_subnet_actor::{ConstructParams, JoinParams, Status};
+use serde::{Deserialize, Serialize};
+
+/// Trait to interact with a subnet and handle its lifecycle.
```

Comment:
nit: Move this stuff to a `subnet` module nested in the `manager` module.

Reply:
You mean moving the `SubnetManager` to a `manager/subnet.rs` file?